packages/datetime/src/timePicker.tsx

```diff
@@ -207,6 +230,41 @@ export class TimePicker extends React.Component<ITimePickerProps, ITimePickerSta
         );
     }
 
+    private maybeRenderAmPm() {
+        if (!this.props.useAmPm) {
+            return null;
+        }
+
+        const onChange = (amPm: string) => {
+            const changeToAm = amPm === "am";
+            if (changeToAm !== this.state.isPm) {
+                return;
+            }
+            const hour = DateUtils.convert24HourMeridiem(this.state.value.getHours(), changeToAm);
+
+            this.setState({ isPm: !changeToAm }, () => {
+                this.updateTime(hour, TimeUnit.HOUR_24);
+            });
+        };
+
+        return (
+            <div className={classNames(CoreClasses.SELECT, Classes.TIMEPICKER_AMPM_SELECT)}>
+                <select
+                    value={this.state.isPm ? "pm" : "am"}
+                    onChange={handleStringChange(onChange)}
+                    disabled={this.props.disabled}
+                >
+                    <option key={0} value={"am"}>
+                        AM
+                    </option>
+                    <option key={1} value={"pm"}>
+                        PM
+                    </option>
```

Comment:
`key` is only necessary when rendering an array. refactor these to: `<option value="pm">PM</option>`